controllers/configurationpolicy_utils.go

```diff
@@ -426,7 +426,7 @@ func createStatus(
 	// Noncompliant with no resources -- return violation immediately
 	if objShouldExist && !compliant && desiredName == "" {
 		message := fmt.Sprintf("No instances of `%v` found as specified", kind)
-		if len(sortedNamespaces) > 0 {
+		if namespaced && len(sortedNamespaces) > 0 {
```

Comment:
Good find! What's happening here, though? Is an empty string being appended to the `sortedNamespaces` array to make its length greater than one?

Reply:
On second thought, that might be a rabbit hole. This will fix the issue--thanks for the fix @willkutler!